docs/prover/cli/options.md

```diff
@@ -379,34 +379,43 @@ Options affecting the type of verification run
 (--coverage_info)=
 ## `coverage_info`
 
-**Usage**
+**Parameters**
 ```sh
---coverage_info <none|basic|advanced>
+none|basic|advanced
 ```
 
 **What does it do?**
-This option enables .sol and .spec coverage analysis and visualization.  The `--coverage_info` option may
-be followed by one of `none`, `basic`, or `advanced`;
+This option enables `.sol` and `.spec` coverage analysis and visualization.
+The `coverage_info` option may be followed by one of `none`, `basic`, or `advanced`;
```

Comment:
should we say that if the flag comes with no value then "basic" is used?

Reply:
added